packages/react-ai/src/components/AIConversation/createProvider.tsx

```diff
@@ -42,32 +48,40 @@ export default function createProvider({
     AIConversationProps,
     'messages' | 'avatars' | 'handleSendMessage' | 'isLoading'
   >): React.JSX.Element {
+    const _displayText = {
+      ...defaultAIConversationDisplayTextEn,
+      ...displayText,
+    };
     return (
-      <ElementsProvider elements={elements}>
-        <ControlsProvider controls={controls}>
-          <SuggestedPromptProvider suggestedPrompts={suggestedPrompts}>
-            <ResponseComponentsProvider responseComponents={responseComponents}>
-              <ConversationInputContextProvider>
-                <SendMessageContextProvider
-                  handleSendMessage={handleSendMessage}
-                >
-                  <AvatarsProvider avatars={avatars}>
-                    <ActionsProvider actions={actions}>
-                      <MessageVariantProvider variant={variant}>
-                        <MessagesProvider messages={messages}>
-                          <LoadingContextProvider isLoading={isLoading}>
-                            {children}
-                          </LoadingContextProvider>
-                        </MessagesProvider>
-                      </MessageVariantProvider>
-                    </ActionsProvider>
-                  </AvatarsProvider>
-                </SendMessageContextProvider>
-              </ConversationInputContextProvider>
-            </ResponseComponentsProvider>
-          </SuggestedPromptProvider>
-        </ControlsProvider>
-      </ElementsProvider>
+      <ConversationDisplayTextProvider {..._displayText}>
```

Comment:
The display text provider should be nested much lower here, ideally below any "static" providers (e.g. `ElementsProvider`

Reply:
Hm can you explain why? I believe you but just curious what is the reasoning

Reply:
Yeah, basically we want any provider that maintains a stable reference to be on the "outside" since they will never cause a re-render

Reply:
Ah yes that makes a lot of sense